discretisedfield/field_new.py

```diff
@@ -327,52 +327,192 @@ def sel(self, *args, **kwargs):
     # mathematical operations
 
     def __abs__(self):
-        raise NotImplementedError()
+        return self.__class__(
+            self.mesh, dim=self.nvdims, value=np.abs(self.data.data), units=self.units
+        )
 
     def __pos__(self):
```

Comment:
This method might work by default, i.e. it might be Python's default behaviour. Removing it and checking if tests pass would be worth doing.

Reply:
@marijanbeg This should be corrected in the latest commit https://github.com/ubermag/discretisedfield/pull/176/commits/2b9b468043c154b21185e61e31f88abe442ca95f

Reply:
Danke :)